cmd/tailscale/backend.go

```diff
@@ -317,15 +319,14 @@ func (b *backend) CloseTUNs() {
 // SetupLogs sets up remote logging.
 func (b *backend) SetupLogs(logDir string, logID logid.PrivateID) {
 	logcfg := logtail.Config{
-		Collection: "tailnode.log.tailscale.io",
-		PrivateID:  logID,
-		Stderr:     log.Writer(),
+		Collection:          logtail.CollectionNode,
+		PrivateID:           logID,
+		Stderr:              log.Writer(),
+		MetricsDelta:        clientmetric.EncodeLogTailMetricsDelta,
+		IncludeProcID:       true,
```

Comment:
Have you been able to compile a local tailscale-debug.apk to try? If not, I can do so.

Reply:
I haven't tried yet, this was a mechanical change and I was relying on CI to make sure nothing breaks.

I'll need to get my Android build set up to [fix up the build tags](https://tailscale.slack.com/archives/C011VBFTHRB/p1681159708507829), I'll hold off merging this until I get that working.

Reply:
Confirmed that it builds and works ([logs](http://logz.ts.tailscale.com:9431/logs?start=2023-04-10T22%3A38%3A23Z&end=2023-04-10T23%3A38%3A23Z&instances=a67162d0f55e3d823cfd427e61b6e389e8a33ec2149f970bff5e6810dd67825f&match=metrics) showing collected metrics).